packages/modules/link-modules/src/definitions/order-promotion.ts

```diff
@@ -44,8 +44,8 @@ export const OrderPromotion: ModuleJoinerConfig = {
       serviceName: Modules.ORDER,
       entity: "Order",
       fieldAlias: {
-        promotion: {
-          path: "promotion_link.promotion",
+        promotions: {
```

Comment:
thought: this is a breaking change; should we keep + deprecate the old alias for now, and remove it later? would like to avoid having to cut a minor for a small change like this

Reply:
like this? https://github.com/medusajs/medusa/pull/12925/files